integration_tests/__init__.py

```diff
@@ -51,9 +53,39 @@ def setUp(self):
         self.useFixture(fixtures.EnvironmentVariable(
             'XDG_CONFIG_HOME', os.path.join(self.path, '.config')))
         self.useFixture(fixtures.EnvironmentVariable(
+            'XDG_CACHE_HOME', os.path.join(self.path, '.cache')))
+        self.useFixture(fixtures.EnvironmentVariable(
             'XDG_DATA_HOME', os.path.join(self.path, 'data')))
         self.useFixture(fixtures.EnvironmentVariable('TERM', 'dumb'))
 
+        patcher = mock.patch(
+            'xdg.BaseDirectory.xdg_config_home',
+            new=os.path.join(self.path, '.config'))
```

Comment:
No, `xdg.BaseDirectory` has those as _variables_! They're evaluated at import time, which is way before `setUp()` runs.